File: src/test/java/cernx/colt/matrix/tdouble/impl/SparseCCDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class SparseCCDoubleMatrix2DViewTest extends SparseCCDoubleMatrix2DTest {

    public SparseCCDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tdouble/impl/SparseCCDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.SparseCCDoubleMatrix2D

class SparseCCDoubleMatrix2DViewTest(arg0: String) extends SparseCCDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseCCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseCCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseCCDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
